projects/ngx-datatable/src/lib/components/body/body.component.ts

```diff
@@ -89,6 +98,7 @@ import { translateXY } from '../../utils/translate';
           >
           </datatable-body-row>
           <ng-template #groupedRowsTemplate>
+            <ng-container>
```

Comment:
Why do we need this ng-container. Does not seem to do anything?

Reply:
Yes. It's not needed. Removed